src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">{`${person.age === undefined ? '' : ` I am ${person.age}`}`}</p>
+    <p className="Person__partner">
+      {person.isMarried && person.sex === 'm' && (
+        `${person.partnerName} is my wife`
+      )}
+      {person.isMarried && person.sex === 'f' && (
```

Comment:
The condition for displaying partner name for female person is not handling the case when partnerName is null or empty. Please add a check for null and empty string values.